contents/data_vis.md

```diff
@@ -368,9 +368,157 @@ What more can we ask for?
 Well, what about different default colors or palettes.
 In our next section, we will see how to use again [Cycles](http://makie.juliaplots.org/stable/theming.html#Cycles) and know a little bit more about them, plus some additional keywords in order to achieve this.
```

Comment:
```suggestion
In our next section, we will see how to use again [`Cycles`](http://makie.juliaplots.org/stable/theming.html#Cycles) and know a little bit more about them, plus some additional keywords in order to achieve this.
```

more consistency since it is a type